.github/workflows/release.yml

```diff
@@ -72,20 +72,55 @@ defaults:
     shell: bash
 
 jobs:
+  release-inputs:
```

Comment:
Determine all package-specific parameters up front in one place. This allows for one action per "function" instead of two with conditionals on the package, which makes job dependencies easier.